docker/softlinkstatic.py

```diff
@@ -7,10 +7,11 @@
 with open(STATIC) as static_fp:
     static_files = json.load(static_fp)
 
-for path in static_files['paths']:
-    full_path = os.path.join('./static', path)
-
-    try:
-        os.unlink(full_path)
-    except OSError:
-        pass
+for orig, hashed in static_files['paths'].items():
+    if '?' in orig:
+        continue
```

Comment:
Before I wrote it this way I validated that in every instance of an `orig` with a query param, there is already a corresponding `orig.split('?')[0]` in `static_files['paths']`, so I didn't see a point in processing those twice.